app/controllers/api/v1/profiles_controller.rb

```diff
@@ -11,6 +11,8 @@ def update
     end
 
     @user.update!(profile_params)
+    @user.custom_attributes.merge!(custom_attributes_params)
+    @user.save!
```

Comment:
avoid multiple saves 

```suggestion
    @user.assign_attributes(profile_params)
    @user.custom_attributes.merge!(custom_attributes_params)
    @user.save!
```